### ISSUE TITLE
Make it possible to login without a 3rd party identity provider

### ISSUE THREAD

**Issue by exyi** [Dzoukr/Srazy.app-Project#2]
I'd much prefer to keep my online identities separated and login with a separate email + password + optional(TOTP). Since I'm apparently a robot, Google and Facebook are always giving me hard time to log in through their platform.

**Comment by Dzoukr** [comment]
I need to think about it. My intention was to use only 3rd party ID and with full respect to GDPR not hold email or any other personal information.

**Comment by exyi** [comment]
If you want to avoid storing emails, you can store a username, password (and TOTP keys). If you want, you can use email for verification, but there is no need to store it (unless you want to be able to send notifications, ...)

Also, you are apparently storing personal names, so things like the "GDPR account delete" have to be possible anyway (it's AFAIK enough to say "contact me, if you want to get deleted")

**Comment by Dzoukr** [comment]
Yup. The whole system is built around GDPR, so deletion (and complete "forgot me") functionality is there. I wanted to say that until making email notification functionality, I'd like to avoid storing any information I don't essentially need. But I'll try to do it together with the email notification feature.
